src/components/dashboard/Dashboard.js

```diff
@@ -42,7 +42,7 @@ const Dashboard = () => {
                 year: 'numeric',
                 day: 'numeric',
               }}
-              events={ events && events.events ? formatEvents(events.events) : []}
+              events={ events ? formatEvents(events) : []}
```

Comment:
I love how this is getting more complicated and cleaner/more readable at the same time 😂